docs/sources/developers/plugins/build-a-logs-data-source-plugin.md

```diff
@@ -4,19 +4,26 @@ title: Build a logs data source plugin
 
 # Build a logs data source plugin
 
-This guide explains how to build a logs data source plugin.
+This guide explains how to build a logs data source plugin. 
 
-Data sources in Grafana supports both metrics and log data. The steps to build a logs data source plugin are largely the same as for a metrics data source. This guide assumes that you're already familiar with how to [Build a data source plugin](/tutorials/build-a-data-source-plugin/) for metrics.
+Data logs are defined as data stored over a period of time in different systems or environments for the purpose of tracking a variety of events.  
```

Comment:
Does it work to say just "events" instead of "a variety of events"?  Does using the word "variety" imply there will be a list of events following? 
```suggestion
Data logs are defined as data stored over a period of time in different systems or environments for the purpose of tracking events.  
```

Reply:
I will check in fix soon, changing to "certain events".

Reply:
Check in fix, available for review.